releases/release-1.29/README.md

```diff
@@ -48,47 +48,46 @@ The 1.29 release cycle is as follows:
+| 1.29.0-beta.0 released                                                                 | Branch Manager                | Thursday 16th November 2023                                                                                           | week 11  |                                                        |
+| Feature blogs ready to review                                                          | Enhancement Owner / SIG Leads | Friday 17th November 2023                                                                                             | week 11  |                                                        |
+| Burndown Meetings daily (Tuesday & Thursday over Slack)                                | Lead                          | Monday 20th November 2023                                                                                             | Week 12  |                                                        |
+| 1.29.0-rc.0 released                                                                   | Branch Manager                | Tuesday 21st November 2023                                                                                            | week 12  |                                                        |
+| release-1.29 branch created                                                            | Branch Manager                | Tuesday 21st November 2023                                                                                            | week 12  |                                                        |
+| release-1.29 jobs created                                                              | Branch Manager                | Tuesday 21st November 2023                                                                                            | week 12  |                                                        |
+| Major Themes complete                                                                  | Release Notes Lead            | Tuesday 28th November 2023                                                                                            | week 13  |                                                        |
+| Docs complete — All PRs reviewed and ready to merge                                    | Docs Lead                     | Tuesday 28th November 2023                                                                                            | week 13  |                                                        |
+| 1.29.0-rc.1 released                                                                   | Branch Manager                | Thursday 28th November 2023                                                                                           | week 13  |                                                        |
```

Comment:
Nov 28 is a Tuesday.
Did you mean Nov 28 or Thursday Nov 30?

Reply:
Good catch! Just pushed a fix (intended 11/28/23)